src/views/Map/Layouts/Desktop/Desktop.tsx

```
@@ -4,29 +4,37 @@ import s from './Desktop.module.css';
   return (
     <div className={s.contentWrap}>
-      <SmartColumn className={s.analytics}>{analyticsColumn}</SmartColumn>
+      {topContent && <div className={s.topContent}>{topContent}</div>}
+      {bottomLeftContent && (
+        <div className={s.bottomLeftContent}>{bottomLeftContent}</div>
+      )}
+      <SmartColumn className={s.leftColumn}>{leftColumn}</SmartColumn>
 
       <div className={s.mapWrap}>
         <div className={s.mapSpaceTop}>{mapColumnTop}</div>
         <div className={s.mapSpaceBlank} id={configRepo.get().mapBlankSpaceId}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)